src/core/core-varlink.c

```diff
@@ -582,6 +567,36 @@ static int manager_varlink_init_user(Manager *m) {
         return 1;
 }
 
+int manager_setup_varlink_server(Manager *m, VarlinkServer **ret_s) {
```

Comment:
This could be made `static`…

Reply:
This is used in the serialization/deserialization code.